contracts/Strategy.sol

```diff
@@ -21,30 +21,25 @@ interface IUni {
 }
 
 
-
-// Import interfaces for many popular DeFi projects, or add your own!
-//import "../interfaces/<protocol>/<Interface>.sol";
-
 contract Strategy is BaseStrategy {
     using SafeERC20 for IERC20;
     using Address for address;
     using SafeMath for uint256;
 
-    ICurveFi public curvePool;// =  ICurveFi(address(0x4CA9b3063Ec5866A4B82E437059D2C43d1be596F));
+    ICurveFi public curvePool;
     ICurveFi public basePool;
-    ICrvV3 public curveToken;// = ICrvV3(address(0xb19059ebb43466C323583928285a49f558E572Fd));
+    ICrvV3 public curveToken;
 
-    address public constant weth = address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
-    address public constant uniswapRouter = address(0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D);
+    address public constant weth = address(0x74b23882a30290451A17c44f4F05243b6b58C76d);
+    address public constant sushiswapRouter = address(0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506);
```

Comment:
personally, I prefer to make constant variables `internal` to save gas, as long as you have no reason to think you'll need to call them externally. 

And there's no need to do `address(0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506);`, you can just do `0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506`. Same applies for all that are just `address`.

Reply:
also, are you actually using sushiswap on fantom? I would imagine spooky swap would have the best liquidity for almost any pair

Reply:
> personally, I prefer to make constant variables `internal` to save gas, as long as you have no reason to think you'll need to call them externally.
> 
> And there's no need to do `address(0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506);`, you can just do `0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506`. Same applies for all that are just `address`.

Fixed

Reply:
> also, are you actually using sushiswap on fantom? I would imagine spooky swap would have the best liquidity for almost any pair

Good callout, changed